pyccel/stdlib/pyc_math/pyc_math.h

```diff
@@ -0,0 +1,10 @@
+#ifndef     PYC_MATH_H
+#define     PYC_MATH_H
+
+long        pyc_factorial(long n);
+long        pyc_gcd (long a, long b);
+long        pyc_lcm (long a, long b);
+double      pyc_radians(double degrees);
+double      pyc_degrees(double radians);
```

Comment:
Is double precision sufficient?

Reply:
See above. We can't pass numpy types to math functions

Reply:
same as a factorial case.